README.md

```diff
@@ -66,6 +66,17 @@ cd scan/wpscan
 bundle install --without test --path vendor/bundle
 </pre>
 
+**Let's encrypt**
+
+Run the letsencrypt-auto file in order for Let's encrypt to install its dependencies. 
+
+<pre>
+cd letsencrypt
+./letsencrypt-auto
```

Comment:
Is it worth complicating the readme though? Might scare our imaginary users away.